src/otx/algorithms/detection/adapters/mmdet/datasets/pipelines/load_pipelines.py

```diff
@@ -84,7 +119,7 @@ def _load_masks(results, ann_info):
 
     def __call__(self, results: Dict[str, Any]):
         """Callback function of LoadAnnotationFromOTXDataset."""
-        dataset_item = results.pop("dataset_item")
+        dataset_item = results.get("dataset_item")
```

Comment:
Could you explain why this should be changed? This will require extra time for deepcoping dataset_item

Reply:
It's because some info in `dataset_item` is used in cache key generation after `LoadAnnotationFromOTXDataset` operation in this new (load img -> load ann -> resize -> cache) operation.
BTW, thank you for the notice. I'll think about other workaround.

Reply:
Addressed in https://github.com/openvinotoolkit/training_extensions/pull/2453/commits/f129c24d9c7c6b95763d405bfb814d104646d8df